cmd/build.go

```diff
@@ -157,7 +170,48 @@ func newBuildCmd() *cobra.Command {
 				return err
 			}
 
-			api := apiNew(entry.APIURL, entry.Token)
+			uuidStr := entry.UUID
+			if uuidStr == "" {
+				fmt.Println("sd-local collects UUIDs for statistical surveys.")
+				fmt.Println("You can reset it later by removing the UUID key from config.")
+				fmt.Print("Would you please cooperate with the survey? [y/N]: ")
+
+				input, err := bufio.NewReader(os.Stdin).ReadString('\n')
+				if err != nil {
+					return err
+				}
+				input = strings.TrimSuffix(input, "\n")
+				if input == "y" || input == "Y" || input == "yes" || input == "Yes" {
+					uuidStr := uuid.NewString()
+
+					err = entry.Set("uuid", uuidStr)
+					if err != nil {
+						return err
+					}
+
+					err = config.Save()
+					if err != nil {
+						return err
+					}
+					fmt.Printf("UUID key has been added to %s\n", configPath)
+				} else {
+					err = entry.Set("uuid", "-")
+					if err != nil {
+						return err
+					}
+					err = config.Save()
+					if err != nil {
+						return err
+					}
+					fmt.Println("UUID key is not set.")
+				}
```

Comment:
This will make it a little more simple, but what do you think?
```suggestion
				uuidStr = "-"
				input = strings.TrimSuffix(input, "\n")
				if input == "y" || input == "Y" || input == "yes" || input == "Yes" {
					uuidStr = uuid.NewString()
				}
				err = entry.Set("uuid", uuidStr)
				if err != nil {
					return err
				}
				err = config.Save()
				if err != nil {
					return err
				}

				if uuidStr != "-" {
					fmt.Printf("UUID key has been added to %s\n", configPath)
				} else {
					fmt.Println("UUID key is not set.")
				}
```